### DISCUSSION TITLE
Disable eslint for a particular project.

### DISCUSSION THREAD

**Discussion by dir1337** [eslint/eslint#17794] [Q&A]
Hello,
Respected devs,

**Problem:**
I am using neovim and installed eslint with mason which is working fine but for temp (sample/demo) files, I don't want to install or make the conifg file of it. Just want to disable it for these temp projects.
Without config it shows error: "No ESLint coniguration found in $PATH/"
How to remove this error without making eslint config file ?

**Comment by Rec0iL99** [comment]
Hi @dir1337, we would need more info to help you here. My wild guess is that you want to ignore certain folders during linting.

https://eslint.org/docs/latest/use/configure/ignore

**Reply by dir1337** [reply]
Yes your guess is "right" but I want eslint to ignore those projects whose root dirs doesn't contain .eslintrc.js or .json config file. 
Is there any option for this feature which automate this process. Otherwise I need to make two file just to test the code snippets and just to get rid of this error (every-time).
Thank you

**Reply by Rec0iL99** [reply]
More info please :). Can you try providing an example?

**Reply by dir1337** [reply]
```
.
├── css
│   └── style.css
├── main.html
├── package.json
└── temp.js
```
This is temp project, where temp.js is a test code file. I don't want to enable eslint for it or don't want to make config file for it. 
Reason : These are the temporary projects and they can be alter or deleted after a time. And there are many of like this, I don't want to make separate eslint config for each project. 
It's a kind of repetitive process for the temporary projects.

**Comment by kecrily** [comment]
Actually, I think it's a problem with the nvim plugin.
